recipes/cuequivariance-jax/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "cuequivariance-jax" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/NVIDIA/cuEquivariance/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 7921940729ea444fa3688907b68056cfb371aba7c47fde231fd4693f19eee5e7
+
+build:
+  skip: true  # [win or py<310]
```

Comment:
Looks like this one could be noarch.

It is OK for noarch package to fails on Windows with a missing dependency. We prefer noarch b/c it is faster to build and the package will be ready when the missing dependency is available on Windows.

Reply:
This is what I did initially, but it made CI fail on Windows. I've removed those commits and all is now noarch (but there will be some windows failures).